app/values/iiif_path.rb

```diff
@@ -9,6 +9,10 @@ def to_s
     "#{iiif_url}/#{escaped_path}"
   end
 
+  def thumbnail
+    "#{self}/full/100,/0/default.jpg"
```

Comment:
`w,` is the [Canonical Syntax](http://iiif.io/api/image/2.1/#canonical-uri-syntax). It's a minor thing since our server supports all of the size options, but this is probably best for interop/code reuse.